bindings/wasm/bindings.js

```diff
@@ -375,18 +582,15 @@ Module.setup = function() {
     return out;
   };
 
-  function batchbool(name) {
-    return function(...args) {
-      if (args.length == 1) args = args[0];
-      const v = new Module.Vector_manifold();
-      for (const m of args) v.push_back(m);
-      const result = Module['_' + name + 'N'](v);
-      v.delete();
-      return result;
-    };
-  }
+  Module.Manifold.prototype = Object.create(ManifoldCtor.prototype);
+
+  // Top-level functions
 
-  Module.union = batchbool('union');
-  Module.difference = batchbool('difference');
-  Module.intersection = batchbool('intersection');
+  Module.triangulate = function(polygons, precision = -1) {
```

Comment:
Should this be on the `CrossSection` prototype instead?

Reply:
The triangulation is mostly used for building meshes though right? Triangulating a polygon doesn't do much good if your goal is using it as a CrossSection.

Reply:
I just mean the input is a CrossSection, not the output. I was imagining something like `const triangles = polygon.triangulate();` instead of `const triangles = Module.triangulate(polygon);`.

Reply:
Ah right. The issue I can think of is that the points Cana change from the users original polygon on making a cross section, so they'll have to make it, put it through triangulate, then get the points from cross section so that they match up with what triangulate worked on. Depends on how low level we want to keep the API for triangulate (and its use case is generally pretty low level, making bespoke meshes).

Reply:
Okay, good point. Let's come back to this another time then.